common/src/View/MapDocument.h

```diff
@@ -237,6 +239,9 @@ namespace TrenchBroom {
         private:
             void doSaveDocument(const IO::Path& path);
             void clearDocument();
+        public: // text encoding
```

Comment:
Should this really be a member function?

Reply:
Yes, it doesn't now, but the encoding should be able to vary per map document (Hexen 2 uses ISO 8859-1) so this is the correct place for it. Longer run we should probably have a status bar popup like text editors do for choosing encoding.

Reply:
Why not just ask the map document for its encoding? It has enough responsibilities as it is I think.